### DISCUSSION TITLE
Splitting Tasks in a separate Menu, Task Mover UI

### DISCUSSION THREAD

**Discussion by rbharathkumar** [jenius-apps/ambie#342] [Ideas]
Hi,
Thank you for building such a wonderful app. Truly enjoy it....
I regularly use the Focus Mode.
I usually add a list of tasks and then start using the Focus Timer.

While using the focus timer in case I am assigned a new task or the current task has led to a new task, it would be nice to be able to add that to the backlog. Currently in the focus mode I am only able to add notes.

One other observation, the UI to move around tasks, would it be better to have the task mover UI element `::` at the left rather than on the right ? I also use Notion and few other apps and used to having it on the left. 

Please note that if these appear to be something specific to me and not applicable to the overall audience, please do ignore it.
Just thought would be nice to let you know.

Thanks you anyways - All the best...
Cheers...
Please

**Comment by dpaulino** [comment]
Good feedback, thanks! I will look into it
